packages/cloud_firestore/cloud_firestore/lib/cloud_firestore.dart

```diff
@@ -4,8 +4,6 @@
 
 library cloud_firestore;
 
-import 'dart:typed_data';
```

Comment:
This probably requires bumping the minimum Flutter SDK version.

To avoid that someone uses flutter <3.3.0 and Firestore fails to compile due to the lack of import on dart:typed_data

Reply:
+1

Reply:
Currently we support `flutter: ">=1.12.13+hotfix.5"`

Reply:
Actually, that might be the better option. Otherwise, this becomes a breaking change. Maybe make a note why we're duplicating import. Also worth checking the example app still builds as well for lower Flutter version 😄

Reply:
For once FVM is useful :p I've checked on lower version, it's okay 👍

Reply:
Soungs good to me.

Reply:
strange why the latest firebase version wasn't giving me this error https://github.com/firebase/flutterfire/issues/9473